operational/vmware/instance_tag_sync/README.md

```diff
@@ -0,0 +1,59 @@
+## VMWare Instance Tag Sync
+
+### What it does
+
+This Policy Template is used to automatically synchronize the CMP Tags to VMWare. 
+
+### Functional Details
+This policy has the following requirements to function properly. You will need to setup a wstunnel to the vsphere server. 
+1. [WS-Tunnel](https://github.com/rightscale/wstunnel)
+1. `/usr/local/bin/wstunnel cli -token $WSTUNNEL_TOKEN -tunnel wss://wstunnel10-1.rightscale.com -server $VSPHERE_HTTPS -logfile /var/log/wstuncli-policy.log -pidfile /var/run/wstunnel-policy.pid`
+
+This policy performs the following action:
+- Synchronizes Instance Tags from RightScale to VMware
```

Comment:
Add a note about 1:1 relationship between policy and single VMWare cloud as a single CMP project could contain multiple VMWare clouds.